lib/elastic_apm/railtie.rb

```diff
@@ -28,14 +28,15 @@ class Railtie < ::Rails::Railtie
     end
 
     initializer 'elastic_apm.initialize' do |app|
-      config = Config.new(app.config.elastic_apm.merge(app: app)).tap do |c|
+      app.config.elastic_apm[:logger] ||= ::Rails.logger
```

Comment:
We need to do this otherwise, a logger object will be created and set on the `config` during the initialization. Then we won't know if we can override it with the `::Rails::logger` or not.